tauri/tauri.conf.json

```diff
@@ -52,7 +59,7 @@
       }
     ],
     "security": {
-      "csp": "default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'"
```

Comment:
The `blob:` loaded worker refuses to load on edge webview in production. Still looking for a solution

Reply:
I am fine disabling CSP for the moment. But when we publish official builds this should be revisited. Can be added to the Tasks list?